example-apps/Token-EasySending/src/web3/account.ts

```diff
@@ -21,6 +21,14 @@ async function isContract(provider: Provider, address: string) : Promise<boolean
     return false;
 }
 
+export function createNameStruct(input: string, type: "email" | "twitter"): NameStruct {
+    return {
+        schema: hash((type == "email" && "mailto").toString()),
```

Comment:
I do not think use "&&" is a good way to determine the logic.
for this case, we do not cover the type = twitter
what if we use something like

```
type = "email" ? "mailto" : "twitter.com"
```

Same for domain and handle

Reply:
Good point, updated